MonoGame.Framework/Graphics/GraphicsDevice.cs

```diff
@@ -1089,8 +1092,49 @@ public void SetVertexBuffer(VertexBuffer vertexBuffer, int vertexOffset)
             _vertexBuffersDirty |= (vertexBuffer == null)
                                    ? _vertexBuffers.Clear()
                                    : _vertexBuffers.Set(vertexBuffer, vertexOffset);
+        }
+
+        /// <inheritdoc cref="SetVertexBuffers(VertexBufferBinding, VertexBufferBinding, VertexBufferBinding, VertexBufferBinding)"/>
+        /// <param name="vertexBuffer">The vertex buffer to set</param>
+        public void SetVertexBuffers(VertexBufferBinding vertexBuffer)
+        {
+            _vertexBufferBindings[0] = vertexBuffer;

```

Comment:
Clear the other elements? (i.e. 1, 2, 3) here and below? Also a prior call to `SetVB(a1, a2, a3)` should clean up after calling `_vertexBuffers.Set` otherwise it's not gonna be GC'ed?